zerver/lib/actions.py

```diff
@@ -4017,7 +4017,7 @@ def do_get_user_invites(user_profile: UserProfile) -> List[Dict[str, Any]]:
     for invitee in prereg_users:
         invites.append(dict(email=invitee.email,
                             ref=invitee.referred_by.email,
-                            invited=invitee.invited_at.strftime("%Y-%m-%d %H:%M:%S"),
+                            invited=datetime_to_timestamp(invitee.invited_at) * 1000,
```

Comment:
the *1000 should be done on the frontend

Reply:
Interesting!! Let me give try!!